addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SpringActuatorScanRule.java

```diff
@@ -196,15 +197,30 @@ private HttpMessage sendActuatorRequest(String encodingType, String actuatorEndp
         return null;
     }
 
-    private void raiseAlert(HttpMessage msg, int confidence, int risk) {
-        newAlert()
-                .setRisk(risk)
-                .setConfidence(confidence)
+    private AlertBuilder createAlert(String evidence) {
+        return newAlert()
+                .setRisk(getRisk())
+                .setConfidence(Alert.CONFIDENCE_MEDIUM)
                 .setName(getAlertName())
-                .setEvidence(msg.getResponseHeader().getPrimeHeader())
                 .setReference(getReference())
-                .setMessage(msg)
-                .setEvidence(StringUtils.left(msg.getResponseBody().toString(), 100))
-                .raise();
+                .setEvidence(evidence);
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        String[] endpointList = {"actuator/health"};
+        String[] encodingTypes = {null, "application/json"};
+        for (String endpoint : endpointList) {
+            for (String encodingType : encodingTypes) {
+                HttpMessage testMsg = sendActuatorRequest(encodingType, endpoint);
+                if (testMsg == null) {
+                    continue;
+                }
+
+                return List.of(
+                    createAlert(StringUtils.left(testMsg.getResponseBody().toString(), 100)).setMessage(testMsg).build()
```

Comment:
The evidence should be hardcoded (and it should not send requests for the example alerts).